app/views/profiles/edit.html+mobile.erb

```diff
@@ -0,0 +1,43 @@
+<div data-controller="profiles" class="w-full pt-2 px-2">
+  <h2 class="text-lg font-bold mb-3">プロフィール編集</h2>
+  <div class="flex mb-4 overflow-scroll">
+    <% @profile.images.each do |profile_image| %>
+      <div class="mr-4 text-center">
+        <%= image_tag profile_image.representation(resize_to_limit: [200, 200]), class: "mb-2" %>
```

Comment:
It may be necessary to conditionally branch if `profile` does not have `images`.
Register a profile without images and access `/profiles`. Then NoMethodError is raised.

Reply:
I'm going to call this method asynchronously using Active Job

https://github.com/kaigionrails/conference-app/blob/dda8cf275d56d626688f9ceec3e8799b0b1b9dde/app/models/profile.rb#L5-L9